XCUITests/FxScreenGraph.swift

```diff
@@ -143,6 +149,7 @@ func createScreenGraph(_ app: XCUIApplication, url: String = "https://www.mozill
     map.createScene(TabTray) { scene in
         scene.tap(app.buttons["TabTrayController.menuButton"], to: TabTrayMenu)
         scene.tap(app.buttons["TabTrayController.addTabButton"], to: NewTabScreen)
+        scene.tap(app.buttons["TabTrayController.maskButton"], to: NewPrivateTab)
```

Comment:
Yes, it makes more sense with the name you suggest. Will do the changes and see how it works. Thanks!